From ebe4ca4f7979762c29fe6050255f4dbfaa8a2280 Mon Sep 17 00:00:00 2001
From: "daniel.tsai" <Tsai.Daniel@inventec.com>
Date: Tue, 6 Jun 2023 02:51:41 +0000
Subject: [PATCH] Add ina220 in PSUSensorMain.cpp

Symptom/Reason :
                 Add ina220 in PSUSensorMain.cpp to enable matching for power monitor.

Root Cause     :
                 N/A

Solution/Change:
                 [PSUSensorMain.cpp]
                         -Add ina220 in pmbusNames and sensorTypes

Entry Test     :
                 N/A
---
 src/PSUSensorMain.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/PSUSensorMain.cpp b/src/PSUSensorMain.cpp
index 7ca5d3d..afd93aa 100644
--- a/src/PSUSensorMain.cpp
+++ b/src/PSUSensorMain.cpp
@@ -50,6 +50,7 @@ static constexpr auto sensorTypes{std::to_array<const char*>(
      "xyz.openbmc_project.Configuration.BMR490",
      "xyz.openbmc_project.Configuration.DPS800",
      "xyz.openbmc_project.Configuration.INA219",
+     "xyz.openbmc_project.Configuration.INA220",
      "xyz.openbmc_project.Configuration.INA230",
      "xyz.openbmc_project.Configuration.IPSPS",
      "xyz.openbmc_project.Configuration.ISL68137",
@@ -86,6 +87,7 @@ static constexpr auto pmbusNames{std::to_array<const char*>({
     "bmr490",
     "dps800",
     "ina219",
+    "ina220",
     "ina230",
     "ipsps1",
     "isl68137",
